src/Aspire.Dashboard/Components/App.razor

```diff
@@ -28,7 +28,5 @@
     <Routes @rendermode="@(new InteractiveServerRenderMode(prerender: false))" />
     <script src="_framework/blazor.web.js"></script>
     <script src="js/app.js"></script>
-    <script src="js/theme.js" type="module"></script>
```

Comment:
Does removing this (presumably leaving it to the module load in OnAfterRenderAsync in MainLayout) cause any delays in theme switching? It feels like there could be a fairly substantial timing delay between when the two loads happen. But maybe it doesn't make a difference.

Reply:
I didn't think of that. Good point. I'll measure.

If I re-add it I'll put a comment explaining why.